otm.py

```diff
@@ -212,6 +212,8 @@ def parse_elf(filename, minimum_size=None, symbol_type_list=None,
         else:
             ppat = None
             npat = None
+            ppati = None
+            ppate = None
```

Comment:
Add these four definitions will make it work.
   
    ppati = None
    ppate = None
    npati = None
    npate = None